src/seedu/addressbook/data/person/Address.java

```diff
@@ -31,4 +31,8 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
     public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
+    
+    public String getPrintableString(){
+    	return "Address: " + this.value;
+    }
```

Comment:
It is a good idea to have the `@Override` comment above this function as well, since it denotes an instance method that overrides a method declared in an interface.